search/management/commands/recreate_index.py

```diff
@@ -41,10 +41,10 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
             import uuid
             profile = cProfile.Profile()
             profile.enable()
-            recreate_index()
+            recreate_index_async.delay()
```

Comment:
A couple things:

- remove the profiling code, it's not doing anything anymore
- print out the task id so if it doesn't finish we can investigate